src/components/FlashWord/FlashWord.module.scss

```diff
@@ -1,17 +1,14 @@
 @import "../../styles/variables";
 
 .word {
```

Comment:
Could you change the color of the word to either $yellow or $blue to make it "pop" out?